packages/flutter/lib/src/rendering/object.dart

```diff
@@ -3130,7 +3130,7 @@ class _RootSemanticsFragment extends _InterestingSemanticsFragment {
   }
 }
 
-/// A [_InterstingSemanticsFragment] that can be told to only add explicit
+/// A [_InterestingSemanticsFragment] that can be told to only add explicit
```

Comment:
I suppose this should be "An [_Interesting...]". I'm confused by the format of the entire file. Private classes with '///' comments?

Reply:
We use `///` for comments that are high enough quality that we could publish them if the class was public, in theory.

Reply:
also, fixed grammar